Documentation/policy/language.rst

```diff
@@ -329,6 +329,10 @@ kube-apiserver
     The kube-apiserver entity represents the kube-apiserver in a Kubernetes
     cluster. This entity represents both deployments of the kube-apiserver:
     within the cluster and outside of the cluster.
+ingress
+    The ingress entity represents the Cilium Envoy instance that handles ingress
+    L7 traffic. Be aware that this also applies for pod-to-pod traffic within
```

Comment:
@sayboras in 1.12 ingress identity doesn't match pod-to-pod traffic because this change haven't been backported to 1.12 (). Therefore, we have to delete the second sentence.